pkg/network/ebpf/c/protocols/postgres/decoding.h

```diff
@@ -70,26 +70,58 @@ static void __always_inline postgres_tcp_termination(conn_tuple_t *tup) {
     bpf_map_delete_elem(&postgres_in_flight, tup);
 }
 
+// Tries to skip the next null-terminated string. Returns the number of bytes to skip, or 0 if the null terminator was
+// not found within the first 128 (POSTGRES_SKIP_STRING_ITERATIONS * BLK_SIZE) bytes.
+static int __always_inline skip_string(pktbuf_t pkt, int message_len) {
+    const __u32 original_data_off = pktbuf_data_offset(pkt);
+    __u32 data_off = original_data_off;
+    __u32 data_end = pktbuf_data_end(pkt);
+    // If the message is larger than the buffer, we limit the data_end to the end of the message.
+    if (data_off + message_len < data_end) {
+        data_end = data_off + message_len;
+    }
+
+    char temp_buffer[BLK_SIZE] = {0};
+    __u8 size_to_read = 0;
+
+    #pragma unroll(POSTGRES_SKIP_STRING_ITERATIONS)
+    for (int iter = 0; iter < POSTGRES_SKIP_STRING_ITERATIONS; iter++) {
```

Comment:
Maybe adding a UT to validate that if we have more than POSTGRES_SKIP_STRING_ITERATIONS we are failing

Reply:
it will be done when we add raw tests

Reply:
we cannot control on the statement string in the parse message without having raw test